examples/technical_blogs/ock_demo_blog/getting_started.md

```diff
@@ -0,0 +1,528 @@
+Getting Started
+===
+
+## Introduction
+
+The release package is a self-contained pre-built project to demonstrate how to run OpenCL
+ and SYCL applications on the RISC-V spike simulator, including a deep neural
+ network demo that uses the VGG network.
+
+We have put together a set of guides that serve as a walkthrough for a selection
+of OpenCL and SYCL applications.
+
+The release package contains a number of different directories:
+
+- oneAPI Construction Kit(OCK) is Codeplay’s multi-target, multi-platform framework.
+    - A few examples have been added to OCK that are useful for analyzing and demonstrating
+      the performance of OCK and oneAPI on RISC-V.
+- portDNN and portBLAS network sample project that executes VGG and Resnet50 networks
+
+```Note: cuda-to-sycl-nbody is in the process of being included to the new OCK demo package```
```

Comment:
scans weird, and I'm not sure you need to say this.

Reply:
Yes, I'll add it once/if sycl_nbody is added to the ock_demo